cmd/apps/istio_app.go

```diff
@@ -114,11 +114,21 @@ func MakeInstallIstio() *cobra.Command {
 		outputPath := path.Join(chartPath, "istio")
 
 		if initIstio, _ := command.Flags().GetBool("init"); initIstio {
-			err = helm3Upgrade(outputPath, "istio/istio-init", namespace, "", defaultVersion, overrides, wait)
+			// Waiting for the crds to appear
+			err = helm3Upgrade(outputPath, "istio/istio-init", namespace, "", defaultVersion, overrides, true)
 			if err != nil {
 				return fmt.Errorf("unable to istio-init install chart with helm %s", err)
 			}
 
+			err = kubectl("-n", "istio-system", "wait", "--for=condition=complete", "job", "--all")
+
+			if err != nil {
+				return fmt.Errorf("error waiting for the crds: %s", err)
+			}
+		}
+
+		if err != nil {
```

Comment:
Why do we have the same error twice, is this a different step?

Reply:
i removed it. copy paste error.